### PR TITLE
Diagram erd

### PR DESCRIPTION
**Pull request by Pamphilemkp**
in this project we created the entity relationship diagram (ERD)  for the vet clinic database using a tool called DRAWSQL .

### CODE REVIEW

File: schema_based_on_diagram.sql [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+  unit_price    decimal,
+  total_price    decimal,
+  quantity      INT,
+  invoice_id     INT,
+  treatment_id  INT,
+
+  CONSTRAINT fk_invoice_id
+      FOREIGN KEY(invoice_id) 
+	  REFERENCES invoices(id)  
+);
```
Comment (1GORDON) [inline schema_based_on_diagram.sql:68]: Great job team 🤝

- [ ] Kindly add a `treatments_id` foreign key to your `invoice_items` table.
- [ ] You may also create an index to this `treatments_id` foreign key.

**Review by 1GORDON — CHANGES_REQUESTED** [review]
##
## Hi @Pamphilemkp & @Piercel2022 ✋

Good job so far!

## Highlights!
- Followed Gitflow ✔️ 
- Good commit messages ✔️
- Linters are passing ✔️
- Descriptive Pull request ✔️

Hello ✋, you have done well reaching this point of the milestone 🤝 . But there are some issues that you still need to work on to go to the next project but you are almost there!

# Status: Required Changes ♻️⛔️

- [ ] Kindly add a Readme file to your project and customize it accordingly. You may use this [readme template](https://github.com/microverseinc/readme-template).

_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you consider them as they can improve your code._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread and tag me `@1GORDON` if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: schema_based_on_diagram.sql (hunk 1 above)
Comment (1GORDON) [inline schema_based_on_diagram.sql:68]: Great job team 🤝. Just like the previous reviewer stated, Kindly implement the changes below and submit your work for approval.

- [ ] Kindly add a `treatments_id` foreign key to your `invoice_items` table.
- [ ] You may also create an index to this `treatments_id` foreign key.

**Review by 1GORDON — CHANGES_REQUESTED** [review]
##
## Hi @Pamphilemkp & @Piercel2022 ✋

Good job so far!

## Highlights!
- Followed Gitflow ✔️ 
- Good commit messages ✔️
- Linters are passing ✔️
- Descriptive Pull request ✔️
- Professional Readme ✔️

Hello ✋, you have done well reaching this point of the milestone 🤝 . But there are some issues that you still need to work on to go to the next project but you are almost there!

# Status: Required Changes ♻️⛔️

_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you consider them as they can improve your code._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread and tag me `@1GORDON` if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: schema_based_on_diagram.sql (hunk 1 above)
Comment (RolandM99) [reply schema_based_on_diagram.sql:68]: - [ ] [**REQUIRED**] As requested by the previous reviewer, make sure that you add `treatments_id` foreign key(FK) to your `invoce_items` table and create an index for it.
take a look at the code snippet below:
```sql
 CONSTRAINT fk_treatment FOREIGN KEY(treatment_id)
    REFERENCES treaments(id)
```
For the index:
```sql
CREATE INDEX invoice_items_treatements_id_index ON invoice_items(id);
```
- Kindly, once implemented the changes requested, ask for an extra review to make your project approved.

**Review by RolandM99 — CHANGES_REQUESTED** [review]
Hi @Pamphilemkp , @Piercel2022 and @PNdunguMaina , 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.


 ## Required Changes ♻️
_Check the comments under the review._
## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by ShadyShawkat — APPROVED** [review]
## Hi Team, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## To Highlight
- Correct use of GitFlow ✔️ 
- Descriptive commit messages ✔️ 
- Professional looking pull request ✔️ 
- Your readme looks professional ✔️ 

Image

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**